tests/test_utils/test_utils.py

```
@@ -0,0 +1,602 @@
+
+    # Detach loop device
+    subprocess.run(["sudo", "losetup", "-d", loop_device], check=True)
+
+    return image_path
+
+
+def create_config(
+    config_path: str,
+    source: str = "",
+    target: str = "",
+    packages: list[str] = [],
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)